### DISCUSSION TITLE
adding a K8s linting tool

### DISCUSSION THREAD

**Discussion by eyarz** [headwaymaps/headway#181] [Ideas]
@ellenhp what is your opinion about adding a K8s linting tool to your CI?
You have already merged [my PR](https://github.com/headwaymaps/headway/pull/165) to fix the K8s manifest, so now they are meeting the basic security standards recommended by the CISA Harding guide.

Adding the linter to the GitHub workflow will ensure the manifests always comply with the Harding guide requirements.
I can help to set it up in the project pipeline and open a PR.

**Comment by ellenhp** [comment]
I like this idea, but I'll need to take a step back from this project very soon due to a conflict of interest with a new employer. Another contributor has mentioned a possible interest in maybe becoming the new maintainer but we'll see what happens there! If this person chooses to take the reigns it'll be up to them whether this is something they want to merge, but I suspect their answer will be yes. More strict linting in CI is (almost?) always a good thing.

If this person doesn't end up taking a maintainer role I'll probably have to archive the project. It will continue to be available but will exist in an essentially frozen state.

**Reply by eyarz** [reply]
got it.
thank you for the update and good luck in your new role!
